packages/language-server/src/plugins/typescript/snapshots/SnapshotManager.ts

```diff
@@ -170,14 +170,12 @@ export class SnapshotManager {
 		this.globalSnapshotsManager.delete(fileName);
 	}
 
-	getFileNames() {
-		return Array.from(this.documents.keys()).map((fileName) => toVirtualFilePath(fileName));
+	getFileNames(): string[] {
+		return Array.from(this.documents.keys());
 	}
 
-	getProjectFileNames() {
-		return this.projectFiles.map((file) => {
-			return toVirtualFilePath(file);
-		});
```

Comment:
So happy to remove this, it was the first nonsense I added to this codebase and now I'm finally removing it ☺️